### PR TITLE
feat: sanity schedule schema

### PR DESCRIPTION
**Pull request by njhuey**
Added schema for schedule

### CODE REVIEW

**Comment by alexanderl19** [comment]
This is all pretty subjective, but

- Was there a reason you chose to have a single schedule document, rather than many event documents?
- Should the description be a Portable Text array, similar to the FAQs?

**Comment by njhuey** [comment]
> This is all pretty subjective, but
> 
>     * Was there a reason you chose to have a single schedule document, rather than many event documents?
> 
>     * Should the description be a Portable Text array, similar to the FAQs?

1) I followed the conventions of the faq schema when designing the singular schedule document. 

2) I skimmed through the previous schedule.json from hackuci 2023 and the description sections consisted of nothing but plain text. I can go ahead and change it to a Portable Text Array if  we want the additional capabilities for this section.

**Comment by alexanderl19** [comment]
I've thought about this for a bit and I think we should change this to individual event documents, rather and one timeline doc for the following reasons:

1. The order in which events are displayed isn't controlled by the user.
2. We may want to query for individual events. (e.g. the next or current event) While this is possible with the current schema, I think it's a bit more intuitive with individual events.
3. We may want to have individual event pages?
4. It's more ergonomic to view and edit the list of events in Sanity Studio.
5. In general, it's probably best to default to using multiple documents and falling back to an array in a document .

File: apps/sanity/schemas/schedule.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { defineType, defineField} from "sanity";
+
+export default defineType({
+	name: "schedule",
```
Comment (alexanderl19) [inline apps/sanity/schemas/schedule.ts:4]: Should this schema be named something like `event` instead?

**Review by alexanderl19 — CHANGES_REQUESTED** [review]

File: apps/sanity/schemas/schedule.ts (hunk 1 above)
Comment (njhuey) [reply apps/sanity/schemas/schedule.ts]: fixed!

File: apps/sanity/schemas/schedule.ts
Comment (alexanderl19) [inline apps/sanity/schemas/schedule.ts:1]: Could you change the file name too?

**Review by alexanderl19 — CHANGES_REQUESTED** [review]

**Review by alexanderl19 — APPROVED** [review]

**Pull request merged**